src/js/api/urls.js

```diff
@@ -69,3 +69,10 @@ export const PUTAWAY_GENERATE_PDF = id => `/putAway/generatePdf/${id}`;
 
 // SUPPORT LINKS
 export const SUPPORT_LINKS = `${CONTEXT_PATH}${API}/supportLinks`;
+
+export const HELPSCOUT_CONFIGURATION = `${CONTEXT_PATH}${API}/helpscout/configuration/`;
```

Comment:
why at the end of this URL is `/`? As I can see we are not using this at the end of other links.

Reply:
This is an external link so it doesn't really matter here as long as the URL works. Some web frameworks (Django for example) prefer (require?) the forward slash at the end. I think it's dumb but there seems to be a semantic reason for it.

https://devpress.csdn.net/python/630453e9c67703293080b33d.html

https://learndjango.com/tutorials/trailing-url-slashes-django

Reply:
One small caveat is that if the external app is expecting a trailing slash and we don't provide one then the app sends a redirect to the browser with the trailing slash. So I don't actually mean "it doesn't matter" as there is overhead but for all intents and purposes it doesn't really matter.

Reply:
@jmiranda I think Alan's point was that we potentially chain a few consts like this:
```js
${API}/${SOMETHING}
```
if `${API}` ended with `/`, we'd have link like: `/api//something` (with two slashes), so it would be better to follow the convention instead of mixing stuff like:
```js
${API}${SOMETHING}
```
and 
```js
${API}/${SOMETHING}
```